app/src/main/java/org/oppia/app/customview/inputInteractionView/NumericInputInteractionView.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.app.customview.inputInteractionView
+
+import android.content.Context
+import android.text.InputFilter
+import android.text.method.DigitsKeyListener
+import android.util.AttributeSet
+import android.widget.EditText
+import org.oppia.app.R
+import org.oppia.app.model.InteractionObject
+
+/** The custom EditText class for numeric input interaction view. */
+class NumericInputInteractionView @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyle: Int = android.R.attr.editTextStyle
+  , private var placeholder: String = "Write the digit here."
+) : EditText(context, attrs, defStyle), InteractionAnswerRetriever {
+  init {
+    attributes()
+  }
+
+  constructor(context: Context?, placeholder: String) : this(context!!) {
```

Comment:
Remove this constructor.
It is safe to remove this constructor now.

Reply:
This is created for dynamically creating the view . In this sample code we are using xml to create them, whereas we would be using this constructor for dynamically creating the view. If you use default constructor you need to pass many more parameters, in order to avoid that this constructor is created.

Reply:
Do we need to dynamically create the view? I wouldn't expect that since we're inflating it using LayoutInflater rather than directly constructing it.

Reply:
As per the interaction model we need to dynamically (programmatically )add view isn’t ?

Reply:
Removed